includes/models/model.llms.lesson.php

```diff
@@ -720,190 +719,216 @@ public function has_content() {
 		}
 	}
 
+
+
+
+
 	/**
 	 * Get next lesson ID
 	 *
 	 * @since 1.0.0
 	 * @since 3.24.0
 	 * @since 4.4.0 Improve query so that unpublished siblings do not break expected results.
-	 * @since [version] Use a numeric comparison for the next position meta query.
+	 * @since [version] Use a numeric comparison for the previous position meta query.
```

Comment:
```suggestion
	 * @since 4.4.2 Use a numeric comparison for the next position meta query.
```

This PR needs to be rebased to the current trunk.